content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -49,9 +56,9 @@ npm install express
 {{< /tabpane >}}
 <!-- prettier-ignore-end -->
 
-### Code
+### Create the sample HTTP Server
 
-Create `app.ts|js` and add the following code to the file:
+Create a file called `app.ts|js` and add the following code to it:
```

Comment:
```suggestion
Create a file named `app.ts|js` and add the following code to it:
```

Reply:
Since this is a beginners guide i think it would be good to avoid potentially confusing syntax like that.

```suggestion
Create a file named `app.ts` (or `app.js` if not using typescript) and add the following code to it:
```

Reply:
Good point, I completely agree.

Reply:
good call @dyladan.

@chalin yes the app should be terminated before moving on